### ISSUE TITLE
Search window sort and misalignment issue

### ISSUE THREAD

**Issue by michelheusschen** [immich-app/immich#7429]
This PR causes that people with name are sorted at the end within the search window.
Additionally, there is vertical missaligment.

![obrazek](https://github.com/immich-app/immich/assets/15554561/1bd6720d-ec8b-4ffa-926c-da0844fb0548)

_Originally posted by @waclaw66 in https://github.com/immich-app/immich/issues/7291#issuecomment-1963910525_

**Comment by michelheusschen** [comment]
Thank you for reporting. The sorting issue is no longer present after #7397 is merged and for the misalignment issue I will create a new PR
